index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>test</title>
```

Comment:
Is this intentional? or do we must title our file as 'tset'? fyi this is a bad start, please fix.

Reply:
Please do changes before set the comment to resolve